buildSrc/src/main/kotlin/nessie-conventions.gradle.kts

```diff
@@ -127,6 +123,19 @@ fun Project.testTasks() {
       if (name != "test") {
         mustRunAfter(tasks.named<Test>("test"))
       }
+
+      if (plugins.withType<QuarkusPlugin>().isNotEmpty()) {
+        jvmArgs("--add-opens=java.base/java.util=ALL-UNNAMED")
+        systemProperty("quarkus.log.level", testLogLevel())
+        systemProperty("quarkus.log.console.level", testLogLevel())
+        systemProperty("http.access.log.level", testLogLevel())
+
+        minHeapSize = if (testHeapSize != null) testHeapSize as String else "512m"
+        maxHeapSize = if (testHeapSize != null) testHeapSize as String else "1536m"
```

Comment:
nit: might be a use-case for https://kotlinlang.org/docs/null-safety.html#elvis-operator